coap-gateway/coap-gw.yaml

```diff
@@ -0,0 +1,68 @@
+log:
+  debug: true
+apis:
+  coap:
+    address: "0.0.0.0:5684"
+    external-port: 5684
+    fqdn: "localhost"
+    request-timeout: 10s
+    heartbeat: 4s
+    max-message-size: 1024  # default:262144
+    keepalive-enabled: true
+    keepalive-timeout: 20s
+    blockwise-transfer-enabled: true
+    blockwise-transfer-block-size: 1024
```

Comment:
move to capabilities section

Reply:
Ok, I got it.  I will move that to capabilities. 
By the way, keepalive and blockwise should be also sub structure as you mentioned below comment of expected coap configuration ? I think capabilities structure is enough to include that.